x/concentrated-liquidity/lp_test.go

```diff
@@ -115,7 +118,7 @@ var (
 			currentTick: DefaultUpperTick + 100,
 
 			preSetChargeSpreadRewards:              sdk.NewDecCoin(ETH, osmomath.ZeroInt()), // zero spread reward
-			expectedSpreadRewardGrowthOutsideLower: oneEthCoins,
+			expectedSpreadRewardGrowthOutsideLower: cl.EmptyCoins,
```

Comment:
Note to reviewer: I think this is a test setup error where this should have been empty before